core/shared/src/main/scala/zio/scheduler/Scheduler.scala

```diff
@@ -16,9 +16,17 @@
 
 package zio.scheduler
 
-import zio.{ Has, ZLayer }
+import zio.{ UIO, ZLayer }
 import zio.internal.{ Scheduler => IScheduler }
 
 object Scheduler extends PlatformSpecific {
-  val live: ZLayer.NoDeps[Nothing, Has[IScheduler]] = ZLayer.succeed(globalScheduler)
+  trait Service extends Serializable {
```

Comment:
I think we need `Scheduler` to be a service so that we can express that the `TestClock` implements it.

Reply:
A service can be any value at all. In the old encoding, it was `IScheduler` (`internal.Scheduler`), i.e. an implementation of that interface. 

Why isn't that enough?